_inc/lib/core-api/class.jetpack-core-api-site-endpoints.php

```diff
@@ -60,4 +61,123 @@ public static function get_features() {
+		}
+
+		if ( Jetpack::is_module_active( 'protect' ) ) {
+			$protect = get_site_option( 'jetpack_protect_blocked_attempts' );
+			if ( $protect > 0 ) {
+				$benefits[] = [
+					'name'        => 'protect',
+					'title'       => 'Brute force protection',
+					'description' => 'The number of malicious login attempts blocked by Jetpack',
+					'value'       => $protect,
+				];
+			}
+		}
+
+		$followers = $stats->stats->followers_blog;
+		if ( $followers > 0 ) {
+			$benefits[] = [
+				'name'        => 'subscribers',
+				'title'       => 'Subscribers',
+				'description' => 'People subscribed to your updates through Jetpack',
+				'value'       => $followers,
+			];
+		}
+
+		$vaultpress = new VaultPress();
+		if ( $vaultpress->is_registered() ) {
+			$data = json_decode( base64_decode( $vaultpress->contact_service( 'plugin_data' ) ) );
+			if ( $data->features->backups && $data->backups->stats->revisions > 0 ) {
+				$benefits[] = [
+					'name'        => 'jetpack-backup',
+					'title'       => 'Jetpack Backup',
+					'description' => 'The number of times Jetpack has backed up your site and kept it safe',
+					'value'       => $data->backups->stats->revisions,
+				];
+			}
+		}
+
+		if ( Jetpack::is_module_active( 'contact-form' ) ) {
+			$contact_form_count = array_sum( get_object_vars( wp_count_posts( 'feedback' ) ) );
+			if ( $contact_form_count > 0 ) {
+				$benefits[] = [
+					'name'        => 'contact-form-feedback',
+					'title'       => 'Contact Form Feedback',
+					'description' => 'Form submissions stored by Jetpack',
+					'value'       => $contact_form_count,
+				];
+			}
+		}
+
+		if ( Jetpack::is_module_active( 'photon' ) ) {
+			$photon_count = $wpdb->get_var( "SELECT COUNT(*) FROM $wpdb->posts WHERE post_mime_type IN ('image/jpeg', 'image/png', 'image/gif')" );
+			if ( $photon_count > 0 ) {
+				$benefits[] = [
+					'name'        => 'image-hosting',
+					'title'       => 'Image Hosting',
+					'description' => 'Super-fast, mobile-ready images served by Jetpack',
+					'value'       => $photon_count,
+				];
+			}
+		}
+
+		$videopress_count = $wpdb->get_var( "SELECT COUNT(*) FROM $wpdb->posts WHERE post_mime_type = 'video/videopress'" );
+		if ( $videopress_count > 0 ) {
+			$benefits[] = [
+				'name'        => 'video-hosting',
+				'title'       => 'Video Hosting',
+				'description' => 'Ad-free, lightning-fast videos delivered by Jetpack',
+				'value'       => $videopress_count,
+			];
+		}
+
+		if ( Jetpack::is_module_active( 'publicize' ) ) {
+			$publicize = new Publicize();
+
+			$connections           = $publicize->get_all_connections();
+			$number_of_connections = count( $connections );
+
+			if ( $number_of_connections > 0 ) {
+				$benefits[] = [
+					'name'        => 'publicize',
+					'title'       => 'Publicize',
+					'description' => 'Live social media site connections, powered by Jetpack',
+					'value'       => count( $connections ),
+				];
+			}
+		}
+
+		if ( $stats->stats->shares > 0 ) {
```

Comment:
This does not seem to work for me in my tests. I always get 0 shares. Do you get results in your tests on Jetpack sites?

Reply:
I thought I had tested it, but I'm testing it now and I'm always getting 0 shares as well. The object returned by the API has properties for shares, but they're all 0. So I think my code is using the API correctly, but it isn't returning the correct number. Should an issue for this go on wpcom or Jetpack?

Reply:
As far as I know this just was never implemented in Jetpack. See #844.

Reply:
Oh that is very good to know. I will follow up in another PR and see if I can get this data another way.

Reply:
We do bump a stat here, so we may have some data maybe, at least for 2 services?
https://github.com/Automattic/jetpack/blob/e4cb8a9302713cf0d853ff42b04a0b7303e79264/modules/sharedaddy/sharing.js#L45

We just don't seem to surface it right now?